test/jdk/com/sun/jdi/TestScaffold.java

```diff
@@ -969,6 +973,8 @@ public static void main(String[] args) throws Throwable {
                     tg.uncaughtThrowable = error;
                 }
             });
+            Thread.currentThread().setName(OLD_MAIN_THREAD_NAME);
```

Comment:
Just want to understand what is this thread naming is about.
Is it just to properly name the threads, or there is some other reason for this?
Also, why OLD_MAIN_THREAD_NAME is a little bit strange and why old?

Reply:
The "main" debuggee thread is always launched as a platform thread. There's no choice in that since it is simply the main application thread. When using the virtual thread wrapper, the "main" thread will spawn a virtual thread to run the debuggee on. By default virtual threads have no name. For the nsk debugger tests, "main" was renamed to "old-m-a-i-n", and the new debuggee virtual thread named "main". For some reason that was never done for the com/sun/jdi tests. I only made this change because in `breakpointReached()` of the test it prints out the thread, and I noticed the name was empty.  I could leave this change out if you want and file a separate CR for it. The test should still run ok without it.

Reply:
Thank you for explaining. Just wanted to understand the motivation.